test/e2e/azure_test.go

```diff
@@ -545,6 +547,7 @@ var _ = Describe("Workload cluster creation", func() {
 						ClusterName:           clusterName,
 						SkipCleanup:           skipCleanup,
 						Windows:               true,
+						IsVMSS:                true,
```

Comment:
could we only disable windows + vmss to limit the impact to the failing test?

Reply:
ok